daml-script/test/daml/upgrades/UpgradesTest.daml

```diff
@@ -106,6 +114,11 @@ tests = mconcat
     , ("Fails if the observers set gets smaller", smallerObserverUpgrade)
     , ("Succeeds if the observer set loses parties that are already signatories", canRemoveObserversThatAreSignatories)
 
+    , -- Ensure clause changes
```

Comment:
Add TODO(#17793): Implement recomputation of ensure clause at upgrade time in engine.